Doc/library/enum.rst

```diff
@@ -55,7 +55,7 @@ helper, :class:`auto`.
 
 .. class:: auto
 
-    Instances are replaced with an appropriate value for Enum members.
+    Instances are replaced with an appropriate value for Enum members.  By default, the initial value starts at 1
```

Comment:
You're missing the trailing period. Sorry for all of the minor fixes.

Reply:
no worries, added the "."
thanks Eric